pinot-common/src/main/java/com/linkedin/pinot/common/data/Schema.java

```diff
@@ -58,13 +58,14 @@
   private final List<DimensionFieldSpec> _dimensionFieldSpecs = new ArrayList<>();
   private final List<MetricFieldSpec> _metricFieldSpecs = new ArrayList<>();
   private TimeFieldSpec _timeFieldSpec;
+
+  // Json ignored fields
   private final Map<String, FieldSpec> _fieldSpecMap = new HashMap<>();
   private final Set<String> _dimensionSet = new HashSet<>();
   private final Set<String> _metricSet = new HashSet<>();
   private final List<String> _dimensionList = new ArrayList<>();
   private final List<String> _metricList = new ArrayList<>();
-
-  private transient String _jsonSchema;
+  private String _jsonSchema;
```

Comment:
I feel it is better to just remove this field, because the call to get JSON schema is not on any critical paths (eg query execution, segment generation).